internal/app/go-filecoin/connectors/storage_miner/connector.go

```diff
@@ -304,11 +326,7 @@ func (m *StorageMinerNodeConnector) GetSealTicket(ctx context.Context, tok stora
 		return storagenode.SealTicket{}, err
 	}
 
-	entropy, err := encoding.Encode(m.minerAddr)
-	if err != nil {
-		return storagenode.SealTicket{}, err
-	}
-	r, err := m.chainState.SampleChainRandomness(ctx, tsk, crypto.DomainSeparationTag_SealRandomness, h-miner.ChainFinalityish, entropy)
+	r, err := m.chainState.SampleChainRandomness(ctx, tsk, crypto.DomainSeparationTag_SealRandomness, h, nil)
```

Comment:
No entropy in the miner-actor, no entropy here.

Reply:
Thanks, please link https://github.com/filecoin-project/specs-actors/issues/260

Reply:
linked